lua/catppuccin/groups/integrations/blink_cmp.lua

```diff
@@ -0,0 +1,41 @@
+local M = {}
+
+function M.get()
+	return {
+		BlinkCmpMenu = { bg = C.base },
+		BlinkCmpMenuBorder = { fg = C.overlay0, bg = C.base },
+		BlinkCmpMenuSelection = { bg = C.surface2 },
```

Comment:
It seems that changing from `surface2` to `surface0` could increase the contrast.